Updated* Add MDX Mandala Exchange Token

gitr0n1n opened the pull request:
https://mandala.exchange

ligi reviewed (changes requested):
Thanks! But to merge it the address must be in [ERC-55](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-55.md) (filename and address field)

gitr0n1n commented:
> Thanks! But to merge it the address must be in [ERC-55](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-55.md) (filename and address field)

Thanks, I converted the filename to the ERC-55 Address and updated the document.
